src/html5/parser/document.rs

```diff
@@ -485,17 +475,38 @@ impl DocumentHandle {
     }
 }
 
+/// This struct will be used to create a fully initialized document or document fragment
+pub struct DocumentBuilder {}
```

Comment:
```suggestion
pub struct DocumentBuilder;
```

(The [unit struct](https://doc.rust-lang.org/book/ch05-01-defining-structs.html#unit-like-structs-without-any-fields).) This can be updated when there are actual fields.